bson/_cbsonmodule.c

```diff
@@ -70,6 +70,11 @@ struct module_state {
     PyObject* _bid_str;
     PyObject* _replace_str;
     PyObject* _astimezone_str;
+    PyObject* _id_str;
+    PyObject* _ref_str;
```

Comment:
Hmm, now the `__id_str` stands out as odd.  Perhaps use `_dollar_<foo>_str` for each of them?

Reply:
good idea! just changed it